pennylane/workflow/qnode.py

```diff
@@ -46,52 +42,48 @@
 SupportedDeviceAPIs = Union["qml.devices.LegacyDevice", "qml.devices.Device"]
 
 
-def _convert_to_interface(res, interface):
+def _convert_to_interface(result, interface):
     """
-    Recursively convert res to the given interface.
+    Recursively convert a result to the given interface.
     """
 
-    if interface == "numpy":
-        return res
+    if interface == Interface.NUMPY:
+        return result
 
-    if isinstance(res, (list, tuple)):
-        return type(res)(_convert_to_interface(r, interface) for r in res)
+    if isinstance(result, (list, tuple)):
+        return type(result)(_convert_to_interface(r, interface) for r in result)
 
-    if isinstance(res, dict):
-        return {k: _convert_to_interface(v, interface) for k, v in res.items()}
+    if isinstance(result, dict):
+        return {k: _convert_to_interface(v, interface) for k, v in result.items()}
 
     interface_conversion_map = {
-        "autograd": "autograd",
-        "jax": "jax",
-        "jax-jit": "jax",
-        "torch": "torch",
-        "tf": "tensorflow",
-        "tf-autograph": "tensorflow",
+        Interface.AUTOGRAD: "autograd",
+        Interface.JAX: "jax",
+        Interface.JAX_JIT: "jax",
+        Interface.TORCH: "torch",
+        Interface.TF: "tensorflow",
+        Interface.TF_AUTOGRAPH: "tensorflow",
     }

```

Comment:
Especially since this is basically the same map for the third time.

Reply:
Resolved in latest commit!